app/controllers/candidate_interface/volunteering/destroy_controller.rb

```diff
@@ -12,10 +12,9 @@ def destroy
         .destroy!
 
       if current_application.application_volunteering_experiences.blank?
-        current_application.update!(volunteering_completed: false, volunteering_experience: false)
+        current_application.update!(volunteering_experience: false)
```

Comment:
Should this come out too?

Reply:
Yep - this will address the issue @paulrobertlloyd mentioned above re: unpaid experience